helpers/class.ItemResolver.php

```diff
@@ -76,7 +80,18 @@ public function resolve($url)
 
         $tmpfile = sys_get_temp_dir() . '/' . md5($url) . '.xml';
         file_put_contents($tmpfile, $dom->saveXML());
-        
+
+        $this->tmpFiles[] = $tmpfile;
+
         return $tmpfile;
     }
+
+    public function __destruct()
+    {
+        foreach ($this->tmpFiles as $tmpFile) {
+            if (file_exists($tmpFile) && is_writable($tmpFile)) {
```

Comment:
```suggestion
            if (is_writable($tmpFile)) {
```
As https://www.php.net/manual/en/function.is-writable.php `Returns true if the filename exists and is writable`